_episodes/14-modules.md

```diff
@@ -6,6 +6,7 @@ questions:
 - "How do we load and unload software packages?"
 objectives:
 - "Understand how to load and use a software package."
+- "Understand how the module mechanism interacts with the shell."
 keypoints:
 - "Load software with `module load softwareName`."
 - "Unload software with `module purge`"
```

Comment:
Note  that module purge would unload all modules, some of which may be default modules loaded upon login.  May want to explain how to unload a single module as well.

Reply:
Replaced `purge` with `unload`, thanks!